### ISSUE TITLE
ValueError

### ISSUE THREAD

**Issue by Shrecker-101** [guilhemmarchand/TA-jira-service-desk-simple-addon#21]
Hi,
running Splunk 8.0.5 in AWS on amz linux

getting when trying to get projects
ValueError at "/opt/splunk/lib/python2.7/json/decoder.py", line 382 : No JSON object could be decoded 
I had to #out owner = admin in default.meta

Any ideas why I am getting this error? 
Thanks

**Comment by guilhemmarchand** [comment]
Hi @Shrecker-101 

Right, You say you had to remove the default admin ownership, to get it working ?

Did you mean once you did you could get the list of projects ?

As well, if you create an alert and add the Jira action, does the project dropdown populates ?

**Comment by Shrecker-101** [comment]
Hi Guilhem,

Apologies for my lack of clarity.
At first there was a dispatchmanager error. This was averted by hashing out owner = admin.

No json object is now the issue.

I have tried to create an alert and no projects are populated.

Kind regards





Charlie



Charlie Shreck



T: +44 (0) 203 972 2532

E: charlie.shreck@redscan.com

W: https://www.redscan.com






________________________________

**Comment by guilhemmarchand** [comment]
Hi @Shrecker-101 

Right, thanks, so:

**1. owner = admin**

By default all objects are owned by the admin user, in the default.meta, which is clean and a good practice but no mandatory.

I would think in your case someone deleted the admin user in your environment, while it's an app fault from the development of view so I will review and maybe remove that out (nobody system would own the objects which I feel less clean but this would address users removing the admin user which isn't very much recommended)

**2. Python error messages during get project, project not populated in the alert UI**

This most certainly indicates that for some reasons your instance cannot access to JIRA, either configuration issue, network constaints or failure, or credentials.

The easiest might be for you to test the connectivity in a shell with a curl command:

**This is in the documentation:**

https://ta-jira-service-desk-simple-addon.readthedocs.io/en/latest/configuration.html

Basically:

`curl -k https://<jira_url>/rest/api/latest/project --user <jira_username>:<jira_password>`

If the curl does not work, then the connectivity in the app will not either.

Let me know the results from the above.

Guilhem

**Comment by guilhemmarchand** [comment]
@Shrecker-101 

I assume the information above replied to your question, please re-open the issue otherwise.

Guilhem
